ui/app/src/components/AuditGridFilterPopover/AuditGridFilterPopoverBody.tsx

```diff
@@ -176,13 +176,11 @@ export function AuditGridFilterPopoverBody(props: AuditGridFilterPopoverProps) {
                 label={<FormattedMessage id="words.from" defaultMessage="From" />}
                 value={fromDate}
                 onChange={onFromDateSelected}
-                renderInput={(props) => <TextField {...props} />}
```

Comment:
https://mui.com/x/migration/migration-pickers-v5/#input-renderer-required-in-v5 now there's no renderInput prop, but slotProps so the field can be configured. Since this was a 'default' renderInput, there is no need to add slotProps